src/virtualship/static/schedule.yaml

```diff
@@ -1,3 +1,12 @@
+area_of_interest:
+  spatial_range:
+    minimum_longitude: -5
+    maximum_longitude: 5
```

Comment:
Do we need to check here that Longitude is either between -180 and 180, or between 0 and 360? Do we need to add a (private) method to the code to make sure this is converted in the way that copernicusmarine expects the longitude range to be? or can that handle both options?